modules/persistent-storage-csi-olm-operator-install.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-aws-efs.adoc
+
+[id="persistent-storage-csi-olm-operator-install_{context}"]
+= Installing the {FeatureName} CSI Operator
+
+The {FeatureName} CSI Operator is not installed in {product-title} by default. Use the following procedure to install and configure this Operator to enable the {FeatureName} CSI Driver Operator in your cluster.
+
+.Prerequisites
+* Access to the {product-title} web console.
+
+.Procedure
+To install the {FeatureName} CSI Operator from the web console:
+
+. Log in to the web console.
+
+. Click *Operators* -> *OperatorHub*.
+
+. Locate the {FeatureName} CSI Operator by typing *{FeatureName} CSI* in the filter box.
+
+. Click the *{FeatureName} CSI Driver Operator* button.
+
+. On the *{FeatureName} CSI Driver Operator* page, click *Install*.
+
+. On the *Install Operator* page, ensure that:
+* *All namespaces on the cluster (default)* is selected.
+* *Installed Namespace* is set to *openshift-cluster-csi-drivers*.
+
+. Click *Install*.
```

Comment:
A note about the community operator would be nice. We want users to install "AWS EFS CSI Driver Operator" and not "
AWS EFS Operator". The second one is not supported by Red Hat!